src/site/views/challenges/newNotify.pug

```diff
@@ -0,0 +1,17 @@
+extends ../layout
+
+block content
+    p(id="data")
+
+        script.
+           var socket = io.connect("#{socketServer}");
```

Comment:
better to create a `.js` file in `./site/public/js` folder, then we will be able to move them into normal application pretty easily, instead of mixing with markup

Reply:
i think so